### ISSUE TITLE
TODO: Owner of files should be root. 

### ISSUE THREAD

**Issue by stpettersens** [stpettersens/gulp-debian#7]
* Owner of files should be root (in group: root).
* Folders should have 755 permissions.
* Scripts should have 755 permissions (done already).
* Normal files should be 644.

Fix this before publishing 0.1.5.

**Comment by lucomsky** [comment]
Now I use this workaround to set root:root for all my files:
`fakeroot gulp`

**Comment by stpettersens** [comment]
Yes, still needs to be corrected. I aim to do this before integrating your PR.

**Comment by stpettersens** [comment]
This should now be done before publishing 0.1.9.
